x-pack/plugins/stack_connectors/common/bedrock/constants.ts

```diff
@@ -23,6 +23,6 @@ export enum SUB_ACTION {
 }
 
 export const DEFAULT_TOKEN_LIMIT = 8191;
-export const DEFAULT_BEDROCK_MODEL = 'anthropic.claude-v2';
+export const DEFAULT_BEDROCK_MODEL = 'anthropic.claude-v2:1';
```

Comment:
@elastic/security-generative-ai LMK if you're OK with this change, happy to revert.